### ISSUE TITLE
WFS tutorial needs update regarding WFS address for GDAL based utilities

### ISSUE THREAD

**Issue by florisvdh** [inbo/tutorials#263]
It seems that current GDAL _needs_ a `WFS:` prefix before the plain `https://` URL ([source](https://gdal.org/drivers/vector/wfs.html#dataset-name-syntax)). Contrary to the GDAL documentation, at least for the BWK WFS (the only case I tested), it appears that '&' should not be changed in our URLs, so the `WFS:` prefix is the only thing to add. (See also a function `getfeature_sf()`, updated accordingly, in https://github.com/r-spatial/sf/issues/1573#issuecomment-866887585.) This affects at least the usecase with `sf::read_sf(url)`, which calls GDAL. I've not tested whether that changes something for the `httr::GET()` approach, but I don't expect so.

While I believe the `request` object below is used only for pasting in the browser:

https://github.com/inbo/tutorials/blob/e16db19ae936631395e842000886def2c71df5c2/content/tutorials/spatial_wfs_services/index.Rmd#L90-L96

... in the following chunks the `request` object is used with `read_sf()` and still lacks the `WFS:` prefix:

https://github.com/inbo/tutorials/blob/e16db19ae936631395e842000886def2c71df5c2/content/tutorials/spatial_wfs_services/index.Rmd#L240-L247

https://github.com/inbo/tutorials/blob/e16db19ae936631395e842000886def2c71df5c2/content/tutorials/spatial_wfs_services/index.Rmd#L272-L283

https://github.com/inbo/tutorials/blob/e16db19ae936631395e842000886def2c71df5c2/content/tutorials/spatial_wfs_services/index.Rmd#L319-L328

https://github.com/inbo/tutorials/blob/e16db19ae936631395e842000886def2c71df5c2/content/tutorials/spatial_wfs_services/index.Rmd#L377-L383

https://github.com/inbo/tutorials/blob/e16db19ae936631395e842000886def2c71df5c2/content/tutorials/spatial_wfs_services/index.Rmd#L573-L583

https://github.com/inbo/tutorials/blob/e16db19ae936631395e842000886def2c71df5c2/content/tutorials/spatial_wfs_services/index.Rmd#L611-L625

Beside the above, there are at least three other chunks where `request` is used for GET - there it must probably stay as-is. So something will need to be said about this difference.

@hansvancalster can you check this further and update the tutorial? Maybe https://github.com/inbo/inborutils/blob/master/R/extract_soil_map_data.R is affected as well, but I don't think so.

**Comment by hansvancalster** [comment]
I ran the code in the tutorial and everything worked. When I load `sf` it is linking to GEOS 3.9.0, GDAL 3.2.1, PROJ 7.2.1. 
The [GDAL documentation](https://gdal.org/drivers/vector/wfs.html#dataset-name-syntax) also states:

> The minimal syntax to open a WFS datasource is : WFS:http://path/to/WFS/service or http://path/to/WFS/service?SERVICE=WFS

So it seems the WFS: prefix is not required. Did you experience trouble without the prefix?

**Comment by florisvdh** [comment]
You're right @hansvancalster, 'today' it runs here as well, without problems :thinking:. Running same versions of the backend libraries. Strangely I thought I had this problem yesterday, but now can't reproduce it anymore - bummer.

We had found that `execGRASS("v.in.ogr")` needs the `WFS:` prefix to work - after that I tried `read_sf()` and it seemed as if it didn't work without `WFS:` either. Most probably that wasn't the cause for error then (perhaps some other URL mistake) - sorry to trouble! Will close this.
